mjcf_to_sdformat/mjcf_to_sdformat/converters/link.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ignition.math import Inertiald, MassMatrix3d, Vector3d, Pose3d
+
+from mjcf_to_sdformat.converters.geometry import (add_mjcf_visual_to_sdf,
+                                                  add_mjcf_collision_to_sdf)
+
+import sdformat as sdf
+
+NUMBER_OF_SDF_LINK = 0
+COLLISION_GEOM_GROUP = 3
+VISUAL_GEOM_GROUP = 0
+
+
+def add_mjcf_geom_to_sdf(geom, inertial):
+    """
+    Converts an MJCF geom to a SDFormat.
+
+    :param mjcf.Element geom: The MJCF geom
+    :param mjcf.Element inertial: Inertial of the body
+    :return: The newly created SDFormat link.
+    :rtype: sdf.Link
+    """
+    link = sdf.Link()
```

Comment:
SDFormat links correspond to MJCF bodies. This seems to associate SDFormat links with MJCF geoms.